### ISSUE TITLE
v0.4.3 fails to build with nightly 2015-08-28

### ISSUE THREAD

**Issue by codyps** [serde-rs/aster#27]
```
/home/cody/.multirust/toolchains/nightly/cargo/registry/src/github.com-0a35038f75765ae4/aster-0.4.3/src/mac.rs:99:13: 99:17 error: structure `syntax::codemap::NameAndSpan` has no field named `name`
/home/cody/.multirust/toolchains/nightly/cargo/registry/src/github.com-0a35038f75765ae4/aster-0.4.3/src/mac.rs:99             name: "test".to_string(),
                                                                                                                              ^~~~
/home/cody/.multirust/toolchains/nightly/cargo/registry/src/github.com-0a35038f75765ae4/aster-0.4.3/src/mac.rs:100:21: 100:44 error: mismatched types:
 expected `syntax::codemap::ExpnFormat`,
    found `fn(syntax::ast::Name) -> syntax::codemap::ExpnFormat {syntax::codemap::ExpnFormat::MacroAttribute}`
(expected enum `syntax::codemap::ExpnFormat`,
    found fn item) [E0308]
/home/cody/.multirust/toolchains/nightly/cargo/registry/src/github.com-0a35038f75765ae4/aster-0.4.3/src/mac.rs:100             format: codemap::MacroAttribute,
                                                                                                                                       ^~~~~~~~~~~~~~~~~~~~~~~
/home/cody/.multirust/toolchains/nightly/cargo/registry/src/github.com-0a35038f75765ae4/aster-0.4.3/src/mac.rs:100:21: 100:44 help: run `rustc --explain E0308` to see a detailed explanation
error: aborting due to 2 previous errors
Could not compile `aster`.
```

**Comment by steveklabnik** [comment]
I am seeing this too.

**Comment by frewsxcv** [comment]
I am seeing this as well

https://travis-ci.org/servo/rust-layers/jobs/77776781

**Comment by bjwbell** [comment]
Aster needs to be updated for https://github.com/rust-lang/rust/pull/28033. I submitted https://github.com/serde-rs/aster/pull/28 to do so.

**Comment by frewsxcv** [comment]
@erickt did 9b64863d07910a7bc9f9fb40c29d26eac80cfd9f fix this?

**Comment by erickt** [comment]
@frewsxcv: Yes it should be fixed in 0.4.4.
